transformations/style_paraphraser/README.md

```diff
@@ -0,0 +1,71 @@
+
+Author name: Filip Cornell
+Author email: c.filip.cornell@gmail.com
+Author affiliation: KTH Royal Institute of Technology
+
+## What type of a transformation is this?
+
+This is a paraphraser that transfers between different styles. Several models are enabled and should be usable through this interface. This therefore provides an easy-to-use interface for these large paraphrasing models, giving often high-quality paraphrases.
+
+The current styles of writing supporting are:
+
+- Shakespeare - A Style transfer paraphraser paraphrasing as if Shakespeare would have written the sentence.
+- Switchboard - A paraphraser trained on a collection of conversational speech transcripts.
+- Tweets - A paraphraser trained on 5.2M English tweets.
+- Bible - A paraphraser trained on texts from the Bible.
+- Romantic poetry - A paraphraser trained on romantic poetry.
+- Basic - A light, basic paraphraser with no specific style.
+
+I will most likely add more pre-trained models as possibilities to use as I add more of the pre-trained models to Huggingface. For more information on these styles, please see the original paper: https://aclanthology.org/2020.emnlp-main.55/.
+
+## What tasks does it intend to benefit?
+Given the multiple different models enabled to be used easily through this interface (Basic, Bible, Shakespeare, Tweets, Switchboard and Romantic poetry), this can be used in a variety of cases. However, this can mainly work as an augmentation for improving sentence classification, perhaps from different domains. As shown by the Krishna K. et al. in their original paper, a diverse paraphrasing has the ability to normalize sentence by removing stylistic identifiers (see Figure 2 in the original paper: https://aclanthology.org/2020.emnlp-main.55/).
+
+## Data and Code Provenance
+This transformation makes use of code and pre-trained models from: https://github.com/martiansideofthemoon/style-transfer-paraphrase. This is therefore entirely based on the paper [Reformulating Unsupervised Style Transfer as Paraphrase Generation
+](https://aclanthology.org/2020.emnlp-main.55/).
+
+The license of the original code is included in the folder, and is an MIT License. Note that the author of this perturbation is not the author of the original code and paper, but has been in direct contact with the main author of the original paper.
+
+## Reference to the original work
+
+The bibtex-reference to the original work is:
+
+```
+@inproceedings{style20,
+author={Kalpesh Krishna and John Wieting and Mohit Iyyer},
+Booktitle = {Empirical Methods in Natural Language Processing},
+Year = "2020",
+Title={Reformulating Unsupervised Style Transfer as Paraphrase Generation},
+}
+```
+
+## What are the limitations of this transformation?
+
+This transformation is limited to different styles of writing in English. No other languages are supported.
+
```

Comment:
From what I observed in the codes, this transformation performs the style transfer sentence by sentence (using sentence segmentation results from NLTK). 

Could you mention here that the contextual information in the input (the style transfer result of one sentence is not affected by other sentences in the input) is not used for the transformation?

Reply:
Mentioned!